config/lib/express.js

```diff
@@ -141,7 +141,7 @@ module.exports.initModulesConfiguration = function (app, db) {
  */
 module.exports.initHelmetHeaders = function (app) {
   // Use helmet to secure Express headers
-  var SIX_MONTHS = 15778476000;
+  var SIX_MONTHS = 15778476;
```

Comment:
FWIW, I prefer having a note that it's in `seconds` here. Even those that understand that Helmet expects seconds, it may be easy to forget or be overlooked by others coming in.

To me it looks less like a "magic" number when I see a comment about `seconds`.